src/commands/google.ts

```diff
@@ -35,8 +36,8 @@ const googleModule: CommandType = {
       })
       .setTimestamp();
 
-    const search_key = "insert custom search api token";
-    const search_id = "insert custom search engine id";
+    const search_key = process.env.GOOGLE_SEARCH_KEY;
+    const search_id = process.env.GOOGLE_SEARCH_ID;
```

Comment:
These can be made available on the `Config` object (see config.ts). This is the preferred way of acquiring configuration values (especially since those which are in the config YAML are not set as environment variables).